### DISCUSSION TITLE
user count for teamtalk

### DISCUSSION THREAD

**Discussion by ahmednaalik51** [BearWare/TeamTalk5#1246] [General]
Hi every one,
In all public server there are a bot called user count that comes in and out. This is a great feature for user to see how many peoples are connected to a teamtalk server, but it can become very anoying to those using a screen reader.
Is there any posible way to make this feature in any other way? Or could you ban in Teamtalk the user bot that can't come back but can still see that the server is activated?
It may anoy people that it has to come in and out of a server so fast, so maybe it could stay connected in the server instead of coming out?
Thoughts?

**Comment by bear101** [comment]
Please see discussion #1247
